topics/scicloud.qmd

```diff
@@ -0,0 +1,48 @@
+---
+title: SciCloud
+---
+## What is it?
+IT for Research (ITvO) offers SciCloud, a service where server capacity can be purchased to support research applications. SciCloud virtual servers are easy to implement, easy to scale up and down and have competitive pricing based on agreed purchase. The virtual server can be placed in an external network so services running on it can be made internet accessible. It can also run on an internal network for extra security and access to some VU services.
```

Comment:
```suggestion
IT for Research (ITvO) offers SciCloud, a service where server capacity can be purchased to support research applications. SciCloud virtual servers are easy to implement, easy to scale up and down and have competitive pricing based on agreed purchase. The virtual server can be placed in an external network, so services running on it can be made internet accessible. It can also run on an internal network for extra security and access to some VU services.
```